pkg/dogstatsd/replay/reader.go

```diff
@@ -35,10 +35,10 @@ type TrafficCaptureReader struct {
 }
 
 // NewTrafficCaptureReader creates a TrafficCaptureReader instance
-func NewTrafficCaptureReader(path string, depth int) (*TrafficCaptureReader, error) {
+func NewTrafficCaptureReader(path string, depth int, mmap bool) (*TrafficCaptureReader, error) {
 
 	// MMap file so that we can have reasonable performance with very large files
-	c, err := getFileMap(path)
+	c, err := getFileMap(path, mmap)
```

Comment:
For clarity it could make sense to rename this function (and update the comment above), since this doesn't necessarily return a memory-mapped file anymore?

Reply:
Another option is to make the test outside `getFileMap`
```
if !mmap {
		c, err = ioutil.ReadFile(path)
} else {
        c, err = getFileMap(path)
}
```

Reply:
Makes sense, I'll update the signature.